README.md

```diff
@@ -78,7 +78,9 @@ versions. Here is the default mapping of versions:
 |:----------------|---------------:|
 | v0.2.6          |         v1.6.0 |
 | v0.5.5          |         v1.7.1 |
-| v1.0.0 - v1.0.3 |         v1.7.1 |
+| v1.0.0 - v1.0.8 |         v1.7.1 |
```

Comment:
Is it acceptable to use `v1.0.z` here instead?  

```suggestion
| v1.0.z |         v1.7.1 |
```

We'll be releasing 1.0 for the life of MTC and I don't foresee the version of Velero changing for it.  In the event it does, we can update this then rather than worry about keeping the range current with each 1.0.z release.